frontend/src/views/RouteFormView.vue

```diff
@@ -4,6 +4,22 @@
       <div class="row">
         <div class="col-lg-8 offset-lg-2">
           <div>
+            <b-alert
```

Comment:
Notifications placed here are never visible when user fills in form on a phone. 
Can we perhaps put it somewhere closer to the button? I am hoping that Weronika's UX ideas will help with this!

Reply:
They are visible but a user needs to scroll up to see it, so I agree we need to find smth more user-friendly from ux perspective.

Reply:
Lets talk on Wednesday